plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/ViewFacet.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext, NX*/
+
+/**
+ * Online status of the repository.
+ *
+ * @since 3.0
+ */
+Ext.define('NX.coreui.view.repository.facet.ViewFacet', {
+  extend: 'Ext.form.FieldContainer',
+  alias: 'widget.nx-coreui-repository-view-facet',
+  requires: [
+    'NX.I18n',
+  ],
+
+  defaults: {
+    xtype: 'textfield'
+  },
+
+  initComponent: function() {
+    var me = this;
+
+    me.items = [
+      {
+        xtype: 'checkbox',
+        name: 'view.online',
+        fieldLabel: 'Online',
```

Comment:
I'm actually more inclined to remove this, at least for now - it isn't actually backed by real data so until it is...